java/sage/SchedulerSelector.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2015 The SageTV Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sage;
+
+public class SchedulerSelector
+{
+  private static final SchedulerInterface ourInstance;
+
+  public static SchedulerInterface getInstance()
+  {
+    return ourInstance;
+  }
+
+  static
+  {
+    // TODO: This will be pulled together in a later commit.
+    if (SeekerSelector.USE_BETA_SEEKER)
+      ourInstance = null; //Scheduler2.getInstance();
+    else
+      ourInstance = Scheduler.getInstance();
```

Comment:
I take this back...I forgot that someone had already convinced me this change was safe. :)  I forgot that the SeekerHolder/SchedulerHolder were using a static variable for these and we were relying on the single threaded nature of the ClassLoader to ensure synchronization and that there creation time wasn't being controlled explicitly anymore. So disregard the two comments I made about this.